onie-image.conf

```diff
@@ -50,3 +50,10 @@ OUTPUT_ABOOT_IMAGE=target/sonic-aboot-$TARGET_MACHINE.swi
 
 ## Aboot boot image name
 ABOOT_BOOT_IMAGE=.sonic-boot.swi
+
+## Output file name for 4-asic mkvm image
+OUTPUT_MKVM_4ASIC_IMAGE=target/sonic-4asic-$TARGET_MACHINE.img
```

Comment:
We could update the other files for other arch's as well, 
onie-image-arm64.conf
onie-image-armhf.conf

Reply:
we can add this later i think.. as we can not verify these environment